src/Provider/GovUkAccount.php

```diff
@@ -451,7 +449,11 @@ protected function createResourceOwner(
         if (!$encoded) {
             throw new \JsonException('Could not encode $response');
         }
-        $response = json_decode($encoded, true);
+
+        $response = json_decode($encoded, true, 512);
```

Comment:
Do we need to define depth?

```suggestion
        $response = json_decode($encoded, true);
```